src/components/story-blocks.js

```diff
@@ -31,7 +33,9 @@ const StoryBlocks = () => {
           bigText={true}
           imageSource={node.image.childImageSharp.fluid}
           altText={node.altText && node.altText}
-          description={node.description && node.description}
+          description={intl.formatMessage({
+            id: `story-blocks.${index}.description`,
```

Comment:
this is correct, but the name is wrong. check line 28, should be stories. again, sorry for the confusing naming, this is something I'd ideally have us clean up to be less confusing in the future but may not be worth the effort for the time we have. Other than that this correct!

Reply:
So here I changed  ` id: `story-blocks.${index}.description`, ` id: `story-blocks.${index}.stories`,`

Reply:
@ToniTepes 'story-blocks' should be stories. You can see the name it should be referencing on line 28